identifiers/country-gb/naw_consitituencies.csv

```diff
@@ -0,0 +1,41 @@
+id,name,gss_code
+ocd-division/country:gb/part:wls/nawc:ynys_mon,Ynys Mon,W09000001

```

Comment:
Instead of part/country-specific types like `nawc`, per discussion in #188, should we just use `ed` (electoral division) for constituencies and `er` (electoral region) for the regions?

If so, please update the readme.

Reply:
Regions and Consituencies are both different types of electoral divisions, so I don't think it makes sense to describe one as "electoral division" and the other one as something else. Lets stick to the words Regions and Consituencies (or abbreviations for them).

Reply:
Agreed, should we use `pcon` and `preg` consistently across all four parts?

Reply:
I've pushed ccf1895 / 7f43e0f where I've gone with gone with `con` and `reg`.

`pcon` is/was **Parliamentary** Constituency. In this PR, we're dealing with migrating the IDs for UK Parl, Scotish Parl and Welsh Assmebly. The Northern Ireland Assembly and Greater London Assembly (also not parliaments) also use constituencies. Lets not add IDs for them here, but in general if you want to say "all constituencies are the same", there's more bodies where constituencies elect to a body that isn't a parliament (or at least the same number of each if you want to say the GLA is different) - lets keep that generic.

Reply:
Great!